.travis.yml

```diff
@@ -1,17 +1,26 @@
 language: c
 sudo: required
-script:
-  - bash -ex .travis-ocaml.sh
-  - make locked-travis
-  - eval $(opam env)
-  - dune external-lib-deps --missing @@default || true
-  - make test
+install: wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-opam.sh
```

Comment:
Here I would have preferred if you made minimal changes to get the CI working again.
I don't think having the CI scripts as submodules or using dune for vendored things broke anything. This new version requires more work to keep the pins in sync with the submodules.

Reply:
I actually wanted to CI to be as standard as possible and close to other projects, and to depend less on vendored scripts.
That being said, I don't think I understand locked enough to get it work the old way.